library/core/core.qs

```diff
@@ -57,5 +57,17 @@ namespace Std.Core {
         output
     }
 
-    export Length, Repeated;
+    /// # Summary
+    /// Represents a complex number by its real and imaginary components.
+    /// The first element of the tuple is the real component,
```

Comment:
Oh just realizing this still refers to the struct as a tuple. Maybe that should be updated?